adapters/ddf-adapter/src/main/java/com/connexta/replication/adapters/ddf/DdfNodeAdapter.java

```diff
@@ -169,9 +169,11 @@ public String getSystemName() {
       return this.cachedSystemName;
     }
 
-    String tagFilter = CqlBuilder.equalTo(Constants.METACARD_TAGS, Constants.REGISTRY_TAG);
-    String identityFilter = CqlBuilder.negate(CqlBuilder.isNull(Constants.REGISTRY_IDENTITY_NODE));
-    String filter = CqlBuilder.allOf(tagFilter, identityFilter);
+    // get any one metacard from the system
+    String modifiedFilter = CqlBuilder.negate(CqlBuilder.isNull(Constants.METACARD_MODIFIED));
+    String tagFilter = CqlBuilder.like(Constants.METACARD_TAGS, "*");
+    String noTagFilter = CqlBuilder.negate(CqlBuilder.isNull(Constants.METACARD_TAGS));
+    String filter = CqlBuilder.anyOf(modifiedFilter, tagFilter, noTagFilter);
```

Comment:
❓ Wouldn't the modified filter be enough to get any metacard?

Reply:
If the system is "new" and doesn't have any resource data in it but does have other metacards like registry, geoname, etc then we can use those as well to get the site id which is why the metacard tags part.